src/seedu/addressbook/storage/jaxb/AdaptedReminder.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.storage.jaxb;
+
+import seedu.addressbook.data.reminder.Reminder;
+
+import javax.xml.bind.annotation.XmlValue;
+
+public class AdaptedReminder {
+
+    @XmlValue
+
+    public String reminderMessage;
+
+    public AdaptedReminder() {}
+
+    public AdaptedReminder(String source) {
+        reminderMessage = source;
+    }
+
+    public Reminder toModelType() {
```

Comment:
This class and its non-trivial methods should have header comments. 
If not, developers who write client code for this class will not know how to use it.